Resources/Prototypes/Entities/Items/Clothing/cloaks.yml

```diff
@@ -0,0 +1,106 @@
+- type: entity
+  parent: Clothing
+  id: CloakClothing
+  abstract: true
+  name: Cloak
+  description:
+  components:
+  - type: Sprite
+    sprite: Clothing/cloak.rsi
+  - type: Icon
+    sprite: Clothing/cloak.rsi
+  - type: Clothing
+    Size: 9999
```

Comment:
What does this do?

Reply:
Make it impossible to be put in backpacks presumably, but also lockers?